app/src/main/java/com/xmartlabs/scasas/criminalintent/ui/crime/list/CrimeAdapter.java

```diff
@@ -67,6 +67,7 @@ public int getItemCount() {
     public CrimeHolder(View itemView, OnTappedListener<Crime> onCrimeTappedListener) {
       super(itemView);
       ButterKnife.bind(this, itemView);
+      solvedCheckBox.setEnabled(false);
```

Comment:
Thanks! Yesterday I was talking of that with mike, because I was looking a way to don't let the user check the crime solved from there but, at the same time, show like it was active anyway. Now with clickable="false" I can achieve that!